recipes/openfoam/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "openfoam" %}
+{% set version = "v2412" %}
+{% set build = 0 %}
```

Comment:
```suggestion
```
Set it directly and don't use an indirection.

Reply:
Set build number directly as suggested